cluster-health-check/cluster-health.sh

```diff
@@ -0,0 +1,519 @@
+#!/bin/bash
+# Minimum space needed to run the script (MB)
+SPACE=2048
+setup() {
+  TMPDIR=$(mktemp -d $MKTEMP_BASEDIR)
+  techo "Created ${TMPDIR}"
+}
+disk-space() {
+  AVAILABLE=$(df -m ${TMPDIR} | tail -n 1 | awk '{ print $4 }')
+  if [ "${AVAILABLE}" -lt "${SPACE}" ]
+    then
+      techo "${AVAILABLE} MB space free, minimum needed is ${SPACE} MB."
+      DISK_FULL=1
+  fi
+}
+verify-access() {
```

Comment:
If the cluster is in a bad state, the `kubectl` command would probably won't go through. It would be good to through an error and bail out than going through the rest of the script.